src/data-types/wrappers.h

```diff
@@ -0,0 +1,326 @@
+#pragma once
+
+#include <errno.h>
+#include <stdarg.h>
+
+#ifdef EOF // stdio.h
```

Comment:
Hi,

I'm getting this all the time:

| vb@fuchur:~/libetpan % git push
| remote: Permission to dinhviethoa/libetpan.git denied to vb@dingens.org.
| fatal: unable to access 'https://github.com/dinhviethoa/libetpan.git/':
| The requested URL returned error: 403
| vb@fuchur:~/libetpan %

No, I didn't switch to dinhviethoa/libetpan.git. It switched
automatically – which gives the error.

Do you know how to prevent that from happening with a pull request?

Yours,
VB.
-- 
Volker Birk
Oberer Graben 4, 8400 Winterthur, Schweiz
mailto:vb@dingens.org  http://fdik.org